include/ableton/platforms/stl/Clock.hpp

```diff
@@ -34,16 +34,14 @@ struct Clock
 
   Clock()
```

Comment:
@llloret Thanks. Could you also remove the constructor, please. There is no use of having it as it does not do anything custom anymore.